Find a functionally equivalent code:android.content.res.TypedArray.getString:COMMENT
Method Modifier: public      
Comment:/**
 * Retrieve the string value for the attribute at <var>index</var>.
 *
 * @param index Index of attribute to retrieve.
 *
 * @return String holding string data.  Any styling information is
 * removed.  Returns null if the attribute is not defined.
 */

@@ -1,4 +1,7 @@
 {
+    if (mRecycled) {
+        throw new RuntimeException("Cannot make calls to a recycled instance!");
+    }
     index *= AssetManager.STYLE_NUM_ENTRIES;
     final int[] data = mData;
     final int type = data[index + AssetManager.STYLE_TYPE];

